templates/tapestry.js

```diff
@@ -1218,15 +1218,15 @@ function tapestryTool(config){
             .append("svg:foreignObject")
             .html(function (d) {
                 return '<i id="mediaButtonIcon' + d.id + '"' + 
-                    ' class="' + getIconClass(d.mediaType, 'play') + ' mediaButtonIcon"' +
+                    ' class="' + ' mediaButtonIcon"' +
                     ' data-id="' + d.id + '"' + 
                     ' data-format="' + d.mediaFormat + '"' + 
                     ' data-media-type="' + d.mediaType + '"' + 
                     ' data-thumb="' + d.imageURL + '"' +
                     ' data-fullscreen="' + d.fullscreen + '"' +
                     ' data-url="' + (d.typeData.mediaURL ? d.typeData.mediaURL : '') + '"' +
                     ' data-media-width="' + d.typeData.mediaWidth + '"' + 
-                    ' data-media-height="' + d.typeData.mediaHeight + '"><\/i>';
+                    ' data-media-height="' + d.typeData.mediaHeight + '">Aa<\/i>';
```

Comment:
I recommend using CSS pseudo-code `:after` to add the `Aa` text in using CSS so that you don't have to add anything in here and it's consistent with how FontAwesome works.